tests/integration/runners/test_fileserver.py

```diff
@@ -161,6 +163,10 @@ def test_file_list(self):
         self.assertIsInstance(ret['return'], list)
         self.assertTrue('grail/scene33' in ret['return'])
 
+    # Git doesn't handle symlinks in Windows. See the thread below:
+    # http://stackoverflow.com/questions/5917249/git-symlinks-in-windows
+    @skipIf(salt.utils.is_windows(),
+            'Git doesn\'t handle symlinks properly on Windows')
```

Comment:
OK, well I still have problems with this then.

1. The skip message doesn't make your explanation clear.
2. If the issue is with how Git for Windows clones repos containing symbolic links, that's not a Salt issue per se.
3. The proper solution isn't to just skip the test because ¯\_(ツ)_/¯, it's to find and fix the underlying issue. The test suite uses the ``roots`` backend, so the relevant code called in that test would be in ``salt/fileserver/roots.py``.